### PR TITLE
Add Play Pagelets to the Module directory

### PR DESCRIPTION
**Pull request by splink**
As Play Pagelets is a kind of framework on top of Play, I have added a new section.

### CODE REVIEW

**Comment by splink** [comment]
btw. I signed the CLA

File: documentation/manual/ModuleDirectory.md [hunk 1]
```diff
@@ -119,6 +119,13 @@ To create your own public module or to migrate from a `play.api.Plugin`, please
 * **Short description:** Allow to package Play! 2.x applications into standard WAR packages.
 
 
+## Frameworks
```
Comment (gmethvin) [inline documentation/manual/ModuleDirectory.md]: Frameworks? Not sure I understand what that means in this context.

Comment (splink) [inline documentation/manual/ModuleDirectory.md]: Can you suggest something else? Pagelets certainly does not fit into any of the existing categories. But I get your point: a framework for a framework, ha?. Other terms that spring to mind would be Structure or Modularization, but I don't think these terms work. In the end I'd call Pagelets a framework. If you want to quickly check out what we're discussing, give it a spin with `activator new
play-pagelets-seed` or have at least quick a look at [a controller which puts it to use.](https://github.com/splink/pagelets-seed/blob/master/app/controllers/HomeController.scala)

Comment (splink) [inline documentation/manual/ModuleDirectory.md]: What do you think about the section title "Extensions" ?

Comment (gmethvin) [inline documentation/manual/ModuleDirectory.md]: How about "Page Rendering"? Unless I don't understand what pagelets does.

**Pull request closed without merging**